src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlTransformer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Xml;
+using System.Xml.Xsl;
+namespace Microsoft.VisualStudio.TestPlatform.Extensions.HtmlLogger
+{
+    class HtmlTransformer : IHtmlTransformer
+    {
+        StringReader xsltStringReader = new StringReader(Resources.Html);
+        XslCompiledTransform myXslTransform;
+
+        /// <summary>
+        /// the following  function invoesthe compiled tranform and Loads the xslt file 
```

Comment:
Comments need to be reviewed. Make sure they are consistent across all the files.